cameraview/src/main/java/com/otaliastudios/cameraview/Camera1.java

```diff
@@ -858,6 +879,20 @@ void setVideoMaxSize(long videoMaxSizeInBytes) {
         mVideoMaxSizeInBytes = videoMaxSizeInBytes;
     }
 
+    @Override
+    void setPlaySounds(boolean playSounds) {
+        final boolean old = mPlaySounds;
+        mPlaySounds = playSounds;
+        schedule(null, true, new Runnable() {
+            @Override
+            public void run() {
+                if (!mergePlaySound(old)) {
+                    mPlaySounds = old;
```

Comment:
This kind of revert is already managed by mergePlaySounds itself. Just call mergePlaySounds() here without any `if`